setup.py

```diff
@@ -1,4 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2003, Taro Ogawa.  All Rights Reserved.
+# Copyright (c) 2013, Savoir-faire Linux inc.  All Rights Reserved.
+
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+# MA 02110-1301 USA
+
 import re
+from io import open
```

Comment:
@null-git I'm importing `open` from `io` so it should work on Python 2.7.

Reply:
I don't think it's required and it's a bit outside of the scope of this MR.
Feel free to create another MR or change this before merge if you feel this is necessary :+1:

Reply:
@pierreduchemin, In my opinion, it's not outside the scope of this PR, the actual issue is #229 and all these changes are trying to fix that. On the last release, we have included some examples with characters that are not UTF-8 on some text files that aren't `*.py` example the README.md. Using this to open these files as UTF-8 is a way to avoid the issue described in #229. I just decided that was a good idea to review all the headers and add the explicit encoding on the python files too.